Ogutu-Brian/Questioner
# #163026478 Setup Directory Structure and Requirements for Questioner back-end(API)

### What does this PR do?
Sets up directory structure and requirements for Questioner API
### Description of Task to be completed?
Have the following achieved
1. The directory structure should allow for scaling of the API
2. The directory structure should adhere to recommended standards
3. Create a file for holding requirements of the project
### How should this be manually tested?
After cloning the repo:
* install tress package
`sudo apt-get install tree`
* `git checkout ch-directory structure-163026478`
* Check the directory structure using tree command
`tree`
### Screenshots
![screenshot from 2019-01-07 23-22-49](https://user-images.githubusercontent.com/30015297/50791662-8e852280-12d3-11e9-9ac4-54a75f45e088.png)

[#163026478](https://www.pivotaltracker.com/story/show/163026478)
